package.json

```diff
@@ -49,7 +49,8 @@
   },
   "homepage": "https://github.com/danny-avila/LibreChat#readme",
   "dependencies": {
-    "axios": "^1.4.0"
+    "axios": "^1.4.0",
+    "puppeteer": "^19.11.1"
```

Comment:
this should not be installed in the root package file, it should be installed from the ./api/ folder.

 you will have to uninstall from the root `npm uninstall langchain` 
 
 then cd `./api/` and install

Reply:
sorry I was half asleep and I wrote the wrong package, edited my comment above. to clarify, I meant just the puppeteer package